lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -1291,6 +1322,33 @@ def test_certificate_custom_language_template_with_locale_language_from_catalogu
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, 'course name: test_right_lang_template')
 
+    @override_settings(FEATURES=FEATURES_WITH_CUSTOM_CERTS_ENABLED)
+    @ddt.data(True, False)
+    @patch('certificates.views.webview.get_course_run_details')
+    @patch('certificates.api.get_course_organization_id')
+    def test_certificate_custom_template_with_hours_of_effort(self, include_effort, mock_get_org_id, mock_get_course_run_details):
```

Comment:
Sanity check: are the `mock_get_org_id` and `mock_get_course_run_details` ordered correctly?

Reply:
Given that the test passes I would say yes. It looks odd, but I believe that is the order that they go in.